### DISCUSSION TITLE
Add Awesome nerd-icons for bufler-list-mode?

### DISCUSSION THREAD

**Discussion by eval-exec** [alphapapa/bufler.el#96] [Ideas]
Hello, do you like nerd-icons? I think it would be cool if `bufler-list-mode` can display nerd-icons in front of "mode name" or "buffer name"

Ref: https://github.com/rainstormstudio/nerd-icons.el

**Comment by alphapapa** [comment]
Hello,

I don't use them, but I've no objection to supporting them.  I don't have time to work on it myself, but feel free to file a feature request and/or send a patch.
